src/domain/user.js

```diff
@@ -183,7 +183,7 @@ export default class User {
     if (key !== undefined && value !== undefined) {
       query.where = {
         profile: {
-          [key]: value
+          [key]: {contains: value}
```

Comment:
Since we didn't have time to discuss this morning, Option 2 may be simplest, using something like this:

```javascript
static async _findMany(key, value, type='equals') {
    const query = {
      include: {
        profile: true
      }
    }

    if (key !== undefined && value !== undefined) {
      query.where = {
        profile: {
          [key]: {
            [type]:value
          }
        }
      }
    }
```

This way, the default behaviour is to do an equals comparison but can be overridden by passing in a type argument - that keeps the method more general purpose whilst still allowing it to work for your use case.

Reply:
I changed this as you have done, but in order for it to work I have to change the find many by first name function to use "contains" which I suppose isnt ideal